src/services/expenses.service.js

```diff
@@ -0,0 +1,85 @@
+let expenses = [];
+const { newId } = require('../utils/newId');
+
+const reset = () => {
+  expenses = [];
+
+  return expenses;
+};
+
+const getAll = ({ from, to, userId, categories }) => {
+  let filteredExpenses = expenses;
+
+  if (from) {
+    filteredExpenses = filteredExpenses.filter((expense) => {
```

Comment:
DRY. Use `filteredExpenses = filteredExpenses.filter((expense) => {` only onde, and move all `if`s inside it